src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service_core.clj

```diff
@@ -123,6 +123,14 @@
         add-servlet-handler (:add-servlet-handler webserver-service)]
     (add-servlet-handler servlet path opts)))
 
+(schema/defn ^:always-validate add-websocket-handler!
+  [context webserver-service
+   svc :- (schema/protocol tk-services/Service)
+   handlers options :- RouteOption]
```

Comment:
So if I'm interpreting this correctly we want https://github.com/richardc/trapperkeeper-webserver-jetty9/commit/6f7617add3a84574d856a5a6b4aa21034668d539 ?